### ISSUE TITLE
Typo in documentation for ContextualEpochGreedyPolicy

### ISSUE THREAD

**Issue by pstansell** [Nth-iteration-labs/contextual#8]
In the documentation from `?ContextualEpochGreedyPolicy` it says, under `Usage`,
```
policy <- EpsilonGreedyPolicy(epsilon = 0.1)
```
which is a different policy, that is, epsilon greedy instead of epoch greedy.

I think the correction would be
```
policy <- ContextualEpochGreedyPolicy $ new (sZl = 0.1)
```

```
sessionInfo()
## R version 3.5.2 (2018-12-20)
## Platform: x86_64-redhat-linux-gnu (64-bit)
## Running under: CentOS release 6.10 (Final)
##
## other attached packages:
## [1] data.table_1.11.4  contextual_0.9.8.2
```

**Comment by robinvanemden** [comment]
Thanks! I will correct this first thing next week.

**Comment by robinvanemden** [comment]
Thanks again for reporting the issue! This has now been fixed in the latest commit, together with #9.
